.github/workflows/build.yaml

```diff
@@ -32,7 +32,10 @@ jobs:
                   case $BUILD_TYPE in
                      "main") export BOOTSTRAP_ARGUMENTS="";;
                      "clang") export BOOTSTRAP_ARGUMENTS="CC=clang-9 CXX=clang++-9";;
-                     "linux-embedded") export BOOTSTRAP_ARGUMENTS="--with-device-layer=linux";;
+                     "linux-embedded")
+                          apt-get update
+                          apt-get install -y libdbus-1-dev
```

Comment:
apt commands should be done in the docker image

Reply:
Yes, can you give a guide on how may I update the `connectedhomeip/chip-build-openssl` image? I've already modified the `Dockerfile`. From the logs in dockerhub it looks like currently @andy31415 is maintaining it?

Reply:
@andy31415 can help out here!

Reply:
Created https://github.com/project-chip/connectedhomeip/pull/1464 for merged docker changes with this and the bloat checker changes for git checkouts.

I will update the image but will not change 'master' yet.

Reply:
The 0.2.18 version worked, thanks for the help.